packages/currency/src/erc20/networks/goerli.ts

```diff
@@ -2,10 +2,10 @@ import { TokenMap } from './types';
 
 // List of the supported goerli ERC20 tokens
 export const supportedGoerliERC20: TokenMap = {
-  // Faucet Token on goerli network. Easy to use on tests.
+  // Faucet Token on goerli network.
   '0xBA62BCfcAaFc6622853cca2BE6Ac7d845BC0f2Dc': {
     decimals: 18,
-    name: 'Faucet Token',
-    symbol: 'FAU-goerli',
+    name: 'FaucetToken',
+    symbol: 'FAU',
```

Comment:
@KolevDarko , do you know if the wrong symbol was on purpose?

Reply:
Not on purpose, probably a copy/paste from currency API because we name them with the network there.